saleor/discount/tasks.py

```diff
@@ -227,12 +227,12 @@ def decrease_voucher_code_usage_of_draft_orders(channel_id: int):
         .filter(code__in=codes)
         .values_list("pk", flat=True)
     )
-    decrease_voucher_codes_usage_task.delay(list(voucher_code_ids))
+    decrease_voucher_codes_usage_task.delay(list(voucher_code_ids), list(codes))
 
 
 @app.task
 @allow_writer()
-def decrease_voucher_codes_usage_task(voucher_code_ids):
+def decrease_voucher_codes_usage_task(voucher_code_ids, codes):
```

Comment:
Just to remind you. If you're going to port this to existing versions, then `code` should be optional. 
(You can have a task in the queue, scheduled by previous implementation, where the new implementation will pick it up).

Reply:
Noted